packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/configurations/ObservableSemantic.java

```diff
@@ -22,12 +24,12 @@ public class ObservableSemantic extends Semantic {
 
 	private String referenceId;
 
-	@Type(JsonType.class)
-	@Column(columnDefinition = "json")
-	private State[] states;
+	private List<String> states;
 
 	private String expression;
 
+	private String expressionMathml;
```

Comment:
I think you will need to sort out camel/snake transforms. I am not exactly sure how we are doing this now so pinging @dvince2

Reply:
Is this being read by an external service? or are they sending us snake-case?

Reply:
I think this should be fine, the new model config is all for our internal use so we can name these pretty much whatever we want and map to the correct values when we create an AMR from config.